### ISSUE TITLE
Submit new profile will not work on Chrome

### ISSUE THREAD

**Issue by bdavidxyz** [joemasilotti/railsdevs.com#781]
## Describe the bug

If you create a new developer profile, then the main form submit will not work on Chrome, because 1) the avatar field is mandatory 2) the avatar field uses createObjectURL, which is not tolerated anymore for a media object on Chrome.

## To reproduce

0. Use Google Chrome
1. Register
2. Click on "I'm looking for work"
3. Fill the whole form, scroll down to 'Save', and click on 'Save'.

## Expected behavior

The profile is successfully created

## Actual behavior

Avatar field is in error. Chrome Console shows  : Failed to execute 'createObjectURL' on 'URL' 

See StackOverflow answer : https://stackoverflow.com/a/53821674/2595513
"This error is caused because the function createObjectURL no longer accepts media stream object for Google Chrome"

Please note that there are no bug when using Firefox.

## Screenshots

I don't have a screenshot, alas

## Additional context

No additional context

**Comment by joemasilotti** [comment]
Thanks for reporting this!

I just ran through in Chrome and was able to create my profile without issue. However, I didn't see the photo preview when I selected one from disk.

Regardless, I'd like to get this fixed. If you (or anyone else) wants to get this fixed then please comment on this issue.

**Comment by ryy** [comment]
Hi, I'm interested in working on the "I didn't see the photo preview when I selected one from disk." issue. Is it still available?

**Comment by joemasilotti** [comment]
Oh, good call @ryy! Can you submit a PR so I can test it out?

**Comment by ryy** [comment]
@joemasilotti 

I'm sorry.
I would like to write a system test but it's not working and is taking a long time. Headless-chrome will not start😱
I'll just post the diffs🙏 

https://github.com/joemasilotti/railsdevs.com/blob/0da13f8fdbb5300eb34b678b33dcee7d79446ea1/app/javascript/controllers/file_upload_controller.js#L7-L13

```diff
  select(event) {
    const file = event.currentTarget.files[0]
    this.imageTarget.src = window.URL.createObjectURL(file)
    this.imageTarget.srcset = window.URL.createObjectURL(file)
+  const picture = event.currentTarget.closest('div').querySelector('picture');
+  if(picture) {
+    const sources = picture.querySelectorAll('source');
+    sources.forEach(source => {
+      source.srcset = window.URL.createObjectURL(file);
+    });
+  }
    this.imageTarget.classList.remove(this.visibilityClass)
    this.errorTarget.classList.add(this.visibilityClass)
  }
```

https://user-images.githubusercontent.com/13917558/219598044-4d97f4fc-eb87-4467-87f4-6c0efd8f0b86.mov

**Comment by joemasilotti** [comment]
@ryy, don't worry about a system test for this.

Can you please submit a PR without it so I can review and merge? Thanks!

**Comment by ryy** [comment]
@joemasilotti 
I've just submitted a [PR](https://github.com/joemasilotti/railsdevs.com/pull/787)!

**Comment by joemasilotti** [comment]
Awesome, thanks! I’ll review soon.

**Comment by joemasilotti** [comment]
Closed via #787.
